### PR TITLE
Update smart wallets demo readme (Resolves WAL-2858)

### PR DESCRIPTION
**Pull request by jmderby**
## Description

Updated smart wallets next.js demo readme auth provider code example as well as release instructions for updating the package.

This pr also fixes the pre-commit hook for Biome to not format md | mdx files. 

## Test plan

readme is looking better than ever 👌 

## Package updates

@crossmint/client-sdk-react-ui

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
         "test": "lerna run test"
     },
     "lint-staged": {
-        "**/*.+(js|json|md|mdx|ts|tsx)": "biome check --write",
```
Comment (devlyn37) [inline package.json:24]: ty!

File: apps/wallets/smart-wallet/next/README.md [hunk 2]
```diff
@@ -112,45 +118,63 @@ Spin up your own instance of the Smart Wallets Demo in under 5 minutes! Below, y
...
+## Process for Updating and Deploying the Next.js Demo
+
+When we release a new version of `@crossmint/client-sdk-react-ui`, we need to update the package in the demo app and deploy it. Here's the process for doing that:
+
+### Instructions
+
+1. Ensure your local `main` branch is up to date
+2. Checkout `smart-wallet-auth-demo-prod`
+3. Update the version of `@crossmint/client-sdk-react-ui` to the latest on npm. (Package link: [npmjs.com/package/@crossmint/client-sdk-react-ui](https://www.npmjs.com/package/@crossmint/client-sdk-react-ui))
+4. Commit the changes and force-push them directly to `smart-wallet-auth-demo-prod`
```
Comment (devlyn37) [inline apps/wallets/smart-wallet/next/README.md:176]: mentioned something in person

Comment (jmderby) [reply apps/wallets/smart-wallet/next/README.md]: updated instruction steps

File: apps/wallets/smart-wallet/next/README.md [hunk 3]
```diff
@@ -94,7 +100,7 @@ Before you begin, ensure you have the following installed:
     ```
 4. In the directory containing this README.md file, rename the `.env.example` file to `.env` and add your API key to the file.
     ```bash
-    NEXT_PUBLIC_CROSSMINT_AUTH_SMART_WALLET_API_KEY="ENTER YOUR CROSSMINT API KEY";
+    NEXT_PUBLIC_CROSSMINT_API_KEY="ENTER YOUR CROSSMINT API KEY";
```
Comment (devlyn37) [inline apps/wallets/smart-wallet/next/README.md:103]: `NEXT_PUBLIC_CROSSMINT_AUTH_SMART_WALLET_API_KEY` is correct here!

Comment (jmderby) [reply apps/wallets/smart-wallet/next/README.md]: good catch! fixed

**Review by devlyn37 — APPROVED** [review]
LGTM

**Pull request merged**